examples/react/contexts/WalletSelectorContext.tsx

```diff
@@ -141,23 +143,25 @@ export const WalletSelectorContextProvider: React.FC<{
       subscription.unsubscribe();
       onHideSubscription.remove();
     };
-  }, [selector]);
+  }, [selector, modal]);
 
-  if (!selector || !modal) {
+  const walletSelectorContextValue = useMemo<WalletSelectorContextValue>(
+    () => ({
+        selector: selector!,
+        modal: modal!,
```

Comment:
using `!` is not idle here, but i guess is ok since it will never pass undefined into the context provider, as it would have been returned from the loading condition, 

or i should change the interface to include `null` support?

Reply:
It's fine either way but I think it's best to change the`WalletSelectorContextValue`  interface.